core/settings_base.py

```diff
@@ -13,9 +13,18 @@
 import os
 from pathlib import Path
 
+
+def gettext(s):
```

Comment:
What does this do? It doesn't seem to be used anywhere.  I tried commenting it out and the code still did the exact same thing